.github/workflows/interface-dependency-versions.yml

```diff
@@ -63,11 +63,11 @@ jobs:
       
       - name: Nightly Catalyst Version
         id: catalyst
-        run: echo "nightly=--extra-index-url https://test.pypi.org/simple/ PennyLane-Catalyst --pre" >> $GITHUB_OUTPUT
+        run: echo "nightly=--index https://test.pypi.org/simple/ --prerelease=allow --upgrade-package PennyLane-Catalyst PennyLane-Catalyst" >> $GITHUB_OUTPUT
```

Comment:
Does `index` replace the pypi.org target, or is it working the same as extra-index-url?
If the former, is there any concern with other dependencies not getting pulled in from where we want (ie the main pypi)?

Reply:
`--index` is the uv equivalent of `--extra-index-url`, as they've deprecated usage of `--extra-index-url` (not sure why).

To replace the pypi.org target, you'd pass `--default-index`